### PR TITLE
Feature/navbar header

### PR DESCRIPTION
**Pull request by MartinPirate**

### CODE REVIEW

**Review by frozenbanana — COMMENTED** [review]
Looks good to me but I will let @mwalbeck do the approval.

File: css/header_nav.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+/* temp background color change */
+#appmenu {
+	background-color: #0f6674 !important;
+}
+
+/* reset logo padding */
+#nextcloud {
+	padding: 0 !important;
```
Comment (mwalbeck) [inline css/header_nav.css:43]: When resetting its more clear to use `unset` instead. For position just below unset is also more preferable as it's more predictable than revert. For both properties !important is unnecessary.

File: css/header_nav.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+/* adjust logo width */
+
+#header .logo {
+	width: 334px;
+}
+
+/* SVG color fill not working */
+#appmenu .app-icon {
```
Comment (mwalbeck) [inline css/header_nav.css:54]: It doesn't work because the icons aren't set up for it, so this can be removed.

File: css/header_nav.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+.header-center span {
+	background-image: url(../img/virtual_assistant.svg);
+	width: 106px;
+	height: 100%;
+	background-repeat: no-repeat;
+	background-position: center;
+}
+
+/* temp background color change */
+#appmenu {
```
Comment (mwalbeck) [inline css/header_nav.css:37]: The height of the header has been increased, but we also need to increase it for the app menu, so it follows along.

The min-width property of `#appmenu` should be increase to 56px as well.

And a rule for `#appmenu li a` should be added increase the height and width to 56px.

File: css/header_nav.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+/* change the logo */
+.logo {
+	background-image: url(../img/logo.png) !important;
+}
+
+/*create the center icon div*/
+.header-center {
+	display: inline-flex;
+	flex: 1;
+	align-items: center;
```
Comment (mwalbeck) [inline css/header_nav.css:24]: If you change this to `justify-content: center;`

And add a rule for `#header .header-right` setting `flex` to 1, then the VA logo should be properly centred on the page.

**Review by mwalbeck — CHANGES_REQUESTED** [review]
The logo and image handling is most likely also going to change, but we don't have the exact method down, so for now that part is good.

File: css/header_nav.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+/* adjust logo width */
+
+#header .logo {
+	width: 334px;
+}
+
+#appmenu li a {
+	min-width: 56px;
```
Comment (mwalbeck) [inline css/header_nav.css:49]: This should just be width, and then also add one for height set to 56px as well.

File: css/header_nav.css (hunk 3 above)
Comment (mwalbeck) [reply css/header_nav.css:32]: We're still missing a min-width property of 56px here

File: css/header_nav.css (hunk 4 above)
Comment (mwalbeck) [reply css/header_nav.css:19]: This still needs doing

**Review by mwalbeck — APPROVED** [review]

**Pull request merged**